Docs/source/building/cori.rst

```diff
@@ -90,36 +92,39 @@ Finally, navigate to the base of the WarpX repository and compile in GPU mode:
     make -j 16 USE_GPU=TRUE
 
 
+.. _building-cori-openPMD:
+
 Building WarpX with openPMD support
 -----------------------------------
 
 First, load the appropriate modules:
 
-::
+.. code-block:: bash
```

Comment:
Why are we doing this change?

Reply:
Because this will add proper console highlighting. Otherwise such code blocks will be guessed or take a default language, which potentially is C++ highlighting.

Reply:
Ah ok ok. That makes sense!